Fix: Kedro-Viz doesn't work when hosted via a URL subpath (#1621)
kedro-org/kedro-viz: RELEASE.md

- Fix dataset factory patterns in Experiment Tracking. (#1588)
- Improved feedback for copy to clipboard feature. (#1614)

> - Fix for Kedro-Viz doesn't work when hosted via a URL subpath (#1621)

Reviewer: ```suggestion
- Ensure Kedro-Viz works when hosted on a URL subpath. (#1621)
```
